packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -1,24 +1,38 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Icon, DesktopWrapper, Money, MobileWrapper, Popover, Text } from '@deriv/components';
+import { Icon, DesktopWrapper, Money, MobileWrapper, Popover, Text, Modal } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import { getCurrencyDisplayCode, getLocalizedBasis } from '@deriv/shared';
 import CancelDealInfo from './cancel-deal-info.jsx';
 
+const PayoutHintModal = ({ is_open, onClose, type }) => (
+    <Modal small is_open={is_open} is_vertical_centered toggleModal={onClose} title={localize('Payout per point')}>
+        <Text size='xxs' as='p' styles={{ padding: '0 1.6rem 1.6rem 1.6rem' }}>
+            <Localize
+                i18n_default_text='<0>For {{title}}:</0> Your payout will grow by this amount for every point {{trade_type}} your strike price.'
+                components={[<strong key={0} />, <br key={1} />]}
+                values={{
+                    trade_type: type === 'VANILLALONGCALL' ? 'above' : 'below',
+                    title: type === 'VANILLALONGCALL' ? 'Call' : 'Put',
```

Comment:
These labels won't be localized inside a variable, you need to localize them inside the condition as well

Reply:
Oh.. I wan't aware of this. Will do it